lib/pushmi_pullyu/cli.rb

```diff
@@ -199,8 +199,9 @@ def rotate_logs
   def run_preservation_cycle
     begin
       entity = queue.wait_next_item
+      # We add + 1 to the get_entity_ingestion_attempt because humans like to start counting from 1
       PushmiPullyu::Logging.log_preservation_attempt(entity,
-                                                     queue.get_entity_ingestion_attempt(entity))
+                                                     queue.get_entity_ingestion_attempt(entity) + 1)
```

Comment:
Small nit: I feel like humanising the counts inside of the log_preservation_x methods would be ideal. Not a show stopper though